specs/themes/ThemeSettings.md

```diff
@@ -0,0 +1,157 @@
+ThemeSettings 
+===
+
+# Background
+_This section is only for internal use, it should not be part of the public documentation._
+
+In UWP, apps can use the
+[AccessibilitySettings](https://learn.microsoft.com/en-us/uwp/api/windows.ui.viewmanagement.accessibilitysettings?view=winrt-22621)
+API to query the system's high-contrast setting, and detect when it changes.  But in a Win32/non-UWP context, the
+**AccessibilitySettings.HighContrastChanged** event never fires.  This is because the event relies on the **CoreWindow**
+object to handle the `WM_THEMECHANGED` message and fire an internal event to the **AccessibilitySettings** object -- but
+in a non-UWP, Win32/WinAppSDK context, no **CoreWindow** exists.
+
+The API described in this doc, **ThemeSettings**, is a WinAppSDK WinRT API that provides the functionality of
+**AccessibilitySettings**, but for a Win32 environment.  We expect to add other APIs to this type over time.
+
+The **ThemeSettings.HighContrast** and **ThemeSettings.HighContrastScheme** properties shown below are basically identical to the
```

Comment:
The HighContrast Boolean seems redundant with HighContrastScheme having a non-empty value. (especially given the below comment that we'll return null/empty rather than the last active Scheme when HC is off.)  Is consistency with the existing APIs valuable in this case?

Reply:
Discussion: What the API enables is to turn HC on and off without having to also pick which HC theme. The Win10 Settings UI had two controls, a toggle and a combo box, this was probably originally designed to match that.  (The API does not guarantee that when false that the string will be null/empty.)

Reply:
Added discussion to background section.